packages/core/src/providers/studio/index.ts

```diff
@@ -186,7 +186,7 @@ export class StudioLivepeerProvider extends BaseLivepeerProvider {
                 metadata: {
                   id: assetId,
                 },
-                ...((source as CreateAssetSourceFile) instanceof File
+                ...((source as unknown as File).name
```

Comment:
I don't think it needs to change, but FTR I believe an alternative would be
```suggestion
                ...(typeof File !== 'undefined' && source instanceof File)
```

That's what I had done in the legacy `video-nft` sdk, just found it: https://github.com/livepeer/video-nft/blob/main/src/api.ts#L229

Reply:
Yeah I like that!